### ISSUE TITLE
ReactStrap DropdownItem onClick type seems wrong

### ISSUE THREAD

**Issue by trichards57** [DefinitelyTyped/DefinitelyTyped#15723]
Hi @albertkim

I am using v4.3.1 of the reactstrap definition file, and I think there is a mistake in the definition for DropdownItem.  You have:

``` typescript
interface Props {
  disabled?: boolean;
  divider?: boolean;
  tag?: React.ReactType;
  header?: boolean;
  onClick?: React.MouseEvent<any>;
  className?: string;
}

declare var DropdownItem: React.StatelessComponent<Props>;
export default DropdownItem;
```

I think it should actually be:

``` typescript
interface Props {
  disabled?: boolean;
  divider?: boolean;
  tag?: React.ReactType;
  header?: boolean;
  onClick?: React.EventHandler<React.MouseEvent<any>>;
  className?: string;
}

declare var DropdownItem: React.StatelessComponent<Props>;
export default DropdownItem;
```
Changing the type of onClick to the EventHandler lets you attach a function to it, which is what the javascript seems to want.

Thanks for all the hard work putting this together :)

**Comment by ddwwcruz** [comment]
I think onClick should be
```ts
onClick?: (event: React.MouseEvent<any>) => void;
```

I see a compiler error otherwise.

**Comment by corydeppen** [comment]
Looks like this was addressed in PR #15926.

**Comment by orta** [comment]
Hi thread, we're moving DefinitelyTyped to use [GitHub Discussions](https://github.com/DefinitelyTyped/DefinitelyTyped/issues/53377) for conversations the `@types` modules in DefinitelyTyped.

To help with the transition, we're closing all issues which haven't had activity in the last 6 months, which includes this issue. If you think closing this issue is a mistake, please pop into the [TypeScript Community Discord](https://discord.gg/typescript) and mention the issue in the `definitely-typed` channel.
